### ISSUE TITLE
hi,

### ISSUE THREAD

**Issue by jacobbogers** [gcanti/functional-programming#24]
dou you have english translation of you fp-ts book?

**Comment by gcanti** [comment]
@jacobbogers alas, no

**Comment by jacobbogers** [comment]
i will try to translate it

**Comment by gcanti** [comment]
That would be awesome

**Comment by DrummerHead** [comment]
@gcanti Does this translation have your blessing?

https://github.com/enricopolanski/functional-programming

**Comment by gcanti** [comment]
@DrummerHead yes and no

- yes because I'm glad someone is willing to make a translation
- no because I have not read it (yet)

(anyway this repo is MIT licensed)

**Comment by DrummerHead** [comment]
Thanks for the answer @gcanti (by the way I already have [this article](https://dev.to/gcanti/functional-design-combinators-14pn) on my todo, thanks for all the teaching material you're creating)

If you could give it a rough look to see if you think it's a proper translation, I'd start reading it. Perhaps [this comment](https://old.reddit.com/r/functionalprogramming/comments/fs9byy/translated_and_edited_translation_of_gcantis/) by the translator can give you more context.

Thanks!

**Comment by enricopolanski** [comment]
@DrummerHead you can check the [CONTRIBUTING](https://github.com/enricopolanski/functional-programming/blob/master/CONTRIBUTING.md) file for my repo. 

TLDR: Structure of the content is the same, laws and definitions do not change, code is Giulio's.

I added some minor changes and few lines more when I felt to add a bit of context in the semigroup sections but there are no major changes. 

I plan in the future to add more content, but as noted in the contribution file, it will mostly be linked externally or as a spoiler in order to preserve the original content.

For now, you can treat the current commit as a 1 : 1.01 translation. So no, it shouldn't have @gcanti's blessing as some content differs and it is a project that I intend to move beyond a translation but, as of this [commit](https://github.com/enricopolanski/functional-programming/tree/9a9938eef5cd2c04aaeb558b3ca3b40a09cfc2f5) treat it as a simple translation.

**Comment by simPod** [comment]
Inglish plz, thiz iz krypto

**Comment by theurgi** [comment]
@enricopolanski  @gcanti 

> In reference to a question about creating a GitBook of the english translation: https://github.com/enricopolanski/functional-programming/issues/24

I would be interested in contributing to convert "Introduction to Functional Programming" into a proper GitBook.

GitBook has [features to support internationalization](https://docs.gitbook.com/tour/internationalization). 

This could also be achieved using any other documentation framework, but consolidating translations into a single repository would make it easier for translators to discover and contribute to this work.

This raises another, broader question about the organization of the [fp-ts](https://github.com/gcanti/fp-ts) project in general which I will ask as a discussion: https://github.com/gcanti/fp-ts/discussions/1802

**Comment by enricopolanski** [comment]
@theurgi I have personally no interest into making it a gitbook, but since it's MIT licensed feel free to fork and do what you prefer.

I think the repo will unavoidably go through some medium size rewriting once the next fp-ts version will be released anyway.
